### DISCUSSION TITLE
Quick Unlock Entry in Windows Hello - how to find and delete?

### DISCUSSION THREAD

**Discussion by gaga9999** [keepassxreboot/keepassxc#11216] [Q&A] [answered]
I accidently used windows hello upon unlocking a database. how do i delete the corresponding entry in windows hello? is my database password stored in windows hello? in credentials manager i cant find a corresponding entry, i have
POP Device entries, Windows LIve but nothing seems to match. Thanks!

**Comment by droidmonkey** [comment, marked as answer]
Just close/quit keepassxc and launch again. Your password is not stored anywhere.

**Comment by gaga9999** [comment]
thanks, how does it work, does it sotre a temporary password for the database?

**Reply by droidmonkey** [reply]
Your database credentials are encrypted using a certificate held by Windows Hello that is stored in your computers TBM chip. The encrypted credentials are stored in KeePassXC's memory and can only be decrypted after authorization to use the certificate. Your fingerprint/pin/etc only authorizes use of the certificate, which is why you have to provide it on unlock first.
